### PR TITLE
Archived sample repos

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[5372efb](https://github.com/OfficeDev/office-js-docs-pr/pull/792/commits/5372efb8da32babd512d180c1b3a2da145d04c25)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[5372efb](https://github.com/OfficeDev/office-js-docs-pr/pull/792/commits/5372efb8da32babd512d180c1b3a2da145d04c25)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[5372efb](https://github.com/OfficeDev/office-js-docs-pr/pull/792/commits/5372efb8da32babd512d180c1b3a2da145d04c25)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/add-in-development-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/concepts/add-in-development-best-practices.md) | :white_check_mark:Succeeded | |
[docs/design/dialog-boxes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/design/dialog-boxes.md) | :white_check_mark:Succeeded | |
[docs/design/task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/design/task-pane-add-ins.md) | :white_check_mark:Succeeded | |
[docs/design/using-office-ui-fabric-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/design/using-office-ui-fabric-react.md) | :white_check_mark:Succeeded | |
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201903010132505322-792%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=6VJw5idvGSED8OiONCbJdcjs1Ey%2B3AGBYqydMlNj%2FwI%3D&st=2019-03-01T01%3A29%3A08Z&se=2019-04-01T01%3A34%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/concepts/add-in-development-best-practices.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Effective add-ins offer unique and compelling functionality that extends Office
 
 - Avoid pop ups. If you have to use them, guide the user to enable your pop up.
 
-For templates that illustrate patterns that you can apply as you develop your first-run experience, see [UX design patterns for Office Add-ins](https://github.com/OfficeDev/Office-Add-in-UX-Design-Patterns-Code).
+For patterns that you can apply as you develop your first-run experience, see [UX design patterns for Office Add-ins](https://docs.microsoft.com/en-us/office/dev/add-ins/design/first-run-experience-patterns).
```
Comment (AlexJerabek) [inline docs/concepts/add-in-development-best-practices.md]: Please remove the localization part of the URL.

File: docs/develop/dialog-api-in-office-add-ins.md [hunk 2]
```diff
@@ -375,7 +373,7 @@ To show a video in a dialog box:
 3.  Use a call of `displayDialogAsync` in the host page to open video.dialogbox.html.
 4.  If your add-in needs to know when the user closes the dialog box, register a handler for the `DialogEventReceived` event and handle the 12006 event. For details, see the section [Errors and events in the dialog window](#errors-and-events-in-the-dialog-window).
 
-For a sample that shows a video in a dialog box, see the [video placemat design pattern](https://github.com/OfficeDev/Office-Add-in-UX-Design-Patterns-Code/tree/master/templates/first-run/video-placemat) in the [UX design patterns for Office Add-ins](https://github.com/OfficeDev/Office-Add-in-UX-Design-Patterns-Code) repo.
+For a sample that shows a video in a dialog box, see the [video placemat design pattern](https://docs.microsoft.com/en-us/office/dev/add-ins/design/first-run-experience-patterns#video-placemat).
```
Comment (AlexJerabek) [inline docs/develop/dialog-api-in-office-add-ins.md]: Please remove the localization part of the URL.

File: docs/design/dialog-boxes.md [hunk 3]
```diff
@@ -27,7 +27,6 @@ For a sample that implements a dialog box, see [Office Add-in Dialog API Example
 
 ## See also
 
-- [GitHub Development Resources](https://github.com/OfficeDev/Office-Add-in-UX-Design-Patterns-Code)
```
Comment (AlexJerabek) [inline docs/design/dialog-boxes.md]: Should we be replacing these with links to the "UX design patterns for Office Add-ins" article?

**Comment by o365devx** [comment]
OPS Build status updates of commit _[dc1ef5c](https://github.com/OfficeDev/office-js-docs-pr/pull/792/commits/dc1ef5c141a1ae34212911a8ce2cfe946dd22839)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[dc1ef5c](https://github.com/OfficeDev/office-js-docs-pr/pull/792/commits/dc1ef5c141a1ae34212911a8ce2cfe946dd22839)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by Rick-Kirkham** [comment]
@AlexJerabek Thanks for the review. Changes made.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[dc1ef5c](https://github.com/OfficeDev/office-js-docs-pr/pull/792/commits/dc1ef5c141a1ae34212911a8ce2cfe946dd22839)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/dialog-boxes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/design/dialog-boxes.md) | :warning:Warning | | [Details](#user-content-docs%2Fdesign%2Fdialog-boxes.md)
[docs/concepts/add-in-development-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/concepts/add-in-development-best-practices.md) | :white_check_mark:Succeeded | |
[docs/design/task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/design/task-pane-add-ins.md) | :white_check_mark:Succeeded | |
[docs/design/using-office-ui-fabric-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/design/using-office-ui-fabric-react.md) | :white_check_mark:Succeeded | |
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |

<a name="docs%2Fdesign%2Fdialog-boxes.md"></a>
### [docs/design/dialog-boxes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/design/dialog-boxes.md)
 - **Line 31**: **[Warning]** Invalid file link:(~/ux-design-pattern-templates.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201903011853328532-792%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=R3J13JlwD%2BBIuM7HmaMy%2BnfaxuNs2savjVKEUb3JiFQ%3D&st=2019-03-01T18%3A49%3A47Z&se=2019-04-01T18%3A54%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[37c61e1](https://github.com/OfficeDev/office-js-docs-pr/pull/792/commits/37c61e1a5639c29398811804c2cf5700a7f7fccb)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[37c61e1](https://github.com/OfficeDev/office-js-docs-pr/pull/792/commits/37c61e1a5639c29398811804c2cf5700a7f7fccb)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[37c61e1](https://github.com/OfficeDev/office-js-docs-pr/pull/792/commits/37c61e1a5639c29398811804c2cf5700a7f7fccb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/add-in-development-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/concepts/add-in-development-best-practices.md) | :white_check_mark:Succeeded | |
[docs/design/dialog-boxes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/design/dialog-boxes.md) | :white_check_mark:Succeeded | |
[docs/design/task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/design/task-pane-add-ins.md) | :white_check_mark:Succeeded | |
[docs/design/using-office-ui-fabric-react.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/design/using-office-ui-fabric-react.md) | :white_check_mark:Succeeded | |
[docs/develop/dialog-api-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/archived-sample-repos/docs/develop/dialog-api-in-office-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201903011952083501-792%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=oygDFRIJ3Ec%2FMuBb5rqacpzio%2FIMSa9CIJZVXygVetU%3D&st=2019-03-01T19%3A48%3A31Z&se=2019-04-01T19%3A53%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**